src/pages/Home/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {Navbar} from "../../components/Home/Navbar/index"
+import {Main} from "../../components/Home/Main/index"
+import {Footer} from "../../components/Home/Footer/index"
+
+export function Home() {
+    return ( 
+        <>
+            <Navbar/>
```

Comment:
Don't use navbar and footer on all page components instead wrap all the pages between the footer and navbar